src/server/components/HtmlLayout.jsx

```diff
@@ -1,13 +1,20 @@
 import { PropTypes } from 'react';
 
 const propTypes = {
-    head: PropTypes.object.isRequired,
+    head: PropTypes.shape({
+        title: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+        meta: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+        link: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+        base: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+        script: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+        htmlAttributes: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
+    }).isRequired,
     style: PropTypes.string.isRequired,
     children: PropTypes.node.isRequired,
 };
 
 const HtmlLayout = ({ head, style, children }) => (
-    <html lang={head.htmlAttributes.lang} {...head.htmlAttributes}>
+    <html lang={head.htmlAttributes.toComponent().lang} {...head.htmlAttributes.toComponent()}>
```

Comment:
Yes indead. It was because of a jsx-a11y rule, but I think it's better to disable it and keep the code nice and tidy